microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -125,57 +133,85 @@ Define the specific markets in which you plan to offer your extension.  The defa
 
 1. Toggle individual markets to exclude each one or select **Unselect all** and then add individual markets of your choice.
 
-   You can change the markets where your extension is offered. A user who installs your extension while it's available in the user's market retains access to your extension. However, the user doesn't have access to any future updates that are submitted to the Microsoft Edge Add-ons website.
+   You can change the markets where your extension is offered.  A user who installs your extension while it's available in the user's market retains access to your extension.  However, the user doesn't have access to any future updates that are submitted to the Microsoft Edge Add-ons website.
+
+1. Click the **Save & Continue** button.
```

Comment:
is **Save & Continue** a button, or link?  Spelled correctly in this article?  
I'm not reg'd w/ Partner Center.

@bhuvanapriyap can you paste a screenshot of Partner Center > **Extension > Properties** webpage?  not found in nearby articles.  If you create screenshot, best to make window as narrow as feasible while still looking realistic - for sharpest result.

& -
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand

rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/publish-extension?branch=pr-en-us-3330#step-4-select-properties-for-your-extension